docs/litmus-operator-ci.yaml

```diff
@@ -32,11 +32,26 @@ metadata:
     app.kubernetes.io/managed-by: kubectl
     name: litmus
 rules:
-- apiGroups: ["","apps","batch","apps.openshift.io","argoproj.io"]
-  resources: ["jobs","deployments","replicationcontrollers","daemonsets","replicasets","statefulsets","deploymentconfigs","rollouts","secrets"]
-  verbs: ["get","list","watch","deletecollection"]
-- apiGroups: ["","litmuschaos.io"]
-  resources: ["pods","configmaps","events","services","chaosengines","chaosexperiments","chaosresults"]
+- apiGroups: [""]
+  resources: ["replicationcontrollers","secrets"]
+  verbs: ["get","list"]
+- apiGroups: ["apps.openshift.io"]
+  resources: ["deploymentconfigs"]
+  verbs: ["get","list"]
+- apiGroups: ["apps"]
+  resources: ["deployments", "daemonsets", "replicasets", "statefulsets"]
+  verbs: ["get","list"]
```

Comment:
deletecollection permission is required for deploymments, daemonsets, jobs and pods for abort case

Reply:
do we really need it for deployments and replicasets @ispeakc0de ? Just realised we need it for daemonsets to support the cases where POD_AFFECTED_PERC is > 0% wherever applicable. 

So I guess we separate out the rule for apps/daemonsets ?

Reply:
In fact, we don't need it on daemonsets either, realized we shifted to launching pods on multiple nodes over daemonsets. So changes are needed on job alone.